algorithm_rgb.py

```diff
@@ -48,15 +48,122 @@
 
```

Comment:
You still need to assign valid strings to the global variables above this line (lint 49)